x11-wm/lumina/lumina-1.2.0.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit qmake-utils
+DESCRIPTION="Lumina desktop environment"
+HOMEPAGE="http://lumina-desktop.org/"
+I18N="161211"
+SRC_URI="https://github.com/trueos/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://dev.gentoo.org/~grozin/${PN}-i18n-${I18N}.tar.bz2"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE=""
+LANGS="af ar az bg bn bs ca cs cy da de el en_GB en_ZA es et eu fa fi fr fr_CA fur gl he hi hr hu id is it ja ka ko lt lv mk mn ms mt nb ne nl pa pl pt pt_BR ro ru sa sk sl sr sv sw ta tg th tr uk ur uz vi zh_CN zh_HK zh_TW zu"
+for lang in ${LANGS}; do
+	IUSE="${IUSE} linguas_${lang}"
```

Comment:
can you please port this from `LINGUAS` to `L10N`
https://wiki.gentoo.org/wiki/Localization/Guide#L10N